app/src/features/skjema-moduler/Inntekt.tsx

```diff
@@ -1,60 +1,127 @@
+import { useQuery, useSuspenseQuery } from "@tanstack/react-query";
 import { format } from "date-fns";
+import { nb } from "date-fns/locale";
 import { Fragment } from "react";
 
-import { inntektQueryOptions } from "~/api/queries.ts";
+import { inntektQueryOptions, personinfoQueryOptions } from "~/api/queries.ts";
+import { InformasjonsseksjonMedKilde } from "~/features/skjema-moduler/PersonOgSelskapsInformasjonSeksjon.tsx";
+import type {
+  ForespørselEntitet,
+  MånedsinntektResponsDto,
+} from "~/types/api-models.ts";
+import { capitalizeSetning, formatKroner, leggTilGenitiv } from "~/utils.ts";
+
+type InntektProps = {
+  forespørsel: ForespørselEntitet;
+};
+export function Inntekt({ forespørsel }: InntektProps) {
+  const førsteDag = capitalizeSetning(
+    format(forespørsel.skjæringstidspunkt, "dd.MM yyyy", {
+      locale: nb,
+    }),
+  );
+
+  const { data: brukerdata } = useSuspenseQuery(
+    personinfoQueryOptions(forespørsel.brukerAktørId, forespørsel.ytelseType),
+  );
 
-export function Inntekt() {
   const inntekt = useQuery(
     inntektQueryOptions({
-      aktorId: "2242003545158",
-      ytelse: "FORELDREPENGER",
-      arbeidsgiverIdent: "896929119",
+      ytelse: forespørsel.ytelseType,
+      aktorId: forespørsel.brukerAktørId,
+      arbeidsgiverIdent: forespørsel.organisasjonsnummer,
+      // arbeidsgiverIdent: "896929119", Bruk disse for å faktisk få data
+      // aktorId: "2242003545158",
       startdato: format(new Date(), "yyyy-MM-dd"),
     }),
   );
 
   return (
-    <>
-      <Heading level="2" size="large">
-        Månedslønn
+    <VStack gap="4">
+      <hr />
+      <Heading level="4" size="medium">
+        Beregnet månedslønn
       </Heading>
-      <VStack gap="2">
-        <HStack justify="space-between">
-          <Label size="small">Utbetalt lønn de siste tre månedene: </Label>
-          <Detail>fra A-ordningen</Detail>
-        </HStack>
-        <HGrid columns={{ md: 2 }} gap="6">
+      <InformasjonsseksjonMedKilde
+        kilde="Fra A-Ordningen"
+        tittel={`${capitalizeSetning(leggTilGenitiv(brukerdata.navn))} lønn fra de siste tre månedene før ${førsteDag}`}
+      >
+        <HGrid columns={{ md: "max-content 1fr" }} gap="4">
           {inntekt.data?.map((inntekt) => (
             <Fragment key={inntekt.fom}>
-              <span>{navnPåMåned(inntekt.fom)}</span>
-              <span>
-                {Intl.NumberFormat("nb-no", {
-                  style: "currency",
-                  currency: "NOK",
-                  maximumFractionDigits: 0,
-                }).format(inntekt.beløp)}
-              </span>
+              <span>{navnPåMåned(inntekt.fom)}:</span>
+              <Label as="span">{formatKroner(inntekt.beløp)}</Label>
             </Fragment>
           ))}
         </HGrid>
+      </InformasjonsseksjonMedKilde>
+      <BodyShort>Beregnet månedslønn</BodyShort>
+      <b>{formatKroner(gjennomsnittInntekt(inntekt.data ?? []))}</b>
```

Comment:
`<strong />` over `<b />`

Reply:
Strong vs b er noe jeg aldri har blitt helt klok på. I mitt hodet er "b" for styling og "strong" for å semantisk fremheve noe. Så da er spørsmålet om kronene her har semantisk mening eller kun styles?